common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -680,14 +687,20 @@ def convert(content, input_format, output_format):
                     content_str = content.decode('latin-1')
             else:
                 content_str = content
-
+            try:
+                content_dict = json.loads(content_str)
+            except ValueError:
+                truncated = content_str[0:min(len(content), 100)].strip()
```

Comment:
You can read the slice operators here as "up to N chars", and the leading 0 is implicit, so you can do:

```python
content_str[:100].strip()
```